clients/client-elastic-beanstalk/src/models/models_0.ts

```diff
@@ -2992,7 +2992,7 @@ export interface DescribeEnvironmentHealthResult {
    *         <code>Updating</code>, <code>Terminating</code>, or <code>Terminated</code>.</p>
    * @public
    */
-  Status?: EnvironmentHealth;
+  Status?: EnvironmentStatus;
```

Comment:
this file is generated from a model data source and not eligible for update from this repository